modules/simple-sa/outputs.tf

```diff
@@ -19,10 +19,21 @@ output "email" {
   value       = google_service_account.sa.email
 }
 
+output "iam_email" {
+  description = "IAM format service account email"
+  value       = "serviceAccount:${google_service_account.sa.email}"
```

Comment:
nit: you can directly use https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account#member

Reply:
Done. Thanks!